extensions/positron-r/src/tasks.ts

```diff
@@ -53,7 +52,9 @@ export async function getRPackageTasks(editorFilePath?: string): Promise<vscode.
 		{
 			task: 'r.task.rmarkdownRender',
 			message: vscode.l10n.t('{taskName}', { taskName: 'Render document with R Markdown' }),
-			rcode: `rmarkdown::render(${editorFilePath})`,
+			// Do not mess with this without exercising it fully on Windows. It's tricky to form
+			// this code snippet in a way that works across OSes.
+			rcode: `rmarkdown::render(''${editorFilePath}'')`,
```

Comment:
This specific use of quotes actually works on Windows (PowerShell). It also works on macOS with a hard-coded path, but not once we interpolate the path 🥲. I will have to leave this as a draft and come back to this.